src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -540,22 +541,39 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     })()
   }, [accountId, accountMetadata, accountType, assetId, bip44Params, chainAdapter, wallet])
 
-  const { data: isTradingActive, refetch: refetchIsTradingActive } = useQuery({
-    ...reactQueries.common.isTradingActive({
-      assetId,
-      swapperName: SwapperName.Thorchain,
-    }),
-    // @lukemorales/query-key-factory only returns queryFn and queryKey - all others will be ignored in the returned object
-    enabled: Boolean(assetId),
+  const { data: inboundAddressesData, refetch: refetchInboundAddressData } = useQuery({
+    ...reactQueries.thornode.inboundAddresses(),
+    enabled: !!assetId,
     // Go stale instantly
     staleTime: 0,
     // Never store queries in cache since we always want fresh data
     gcTime: 0,
     refetchOnWindowFocus: true,
     refetchOnMount: true,
     refetchInterval: 60_000,
+    select: data => selectInboundAddressData(data, assetId),
   })
 
+  const {
+    data: mimir,
+    isLoading: isMimirLoading,
+    refetch: refetchMimir,
+  } = useQuery({
+    ...reactQueries.thornode.mimir(),
+    staleTime: thorchainBlockTimeMs,
+  })
```

Comment:
all this repeat `useQuery` then `selectIsTradingActive` is begging for a hook to simplify things, a bit like the swapper `useIsTradingActive` but not 2 asset combined in 1 hook. In fact you could just re-use the new hook 2 times for swapper and delete the old hook. Is that reasonable or feasible to do?

Reply:
gm e5e3c825dd844a0e8d9b1cb729e543bc212a637e